.gitignore

```diff
@@ -79,7 +79,7 @@ loolmap
 loolconvert
 loolsocketdump
 loolstress
-loolconfig
```

Comment:
Would be good to still keep loolconfig, but only as a symlink to coolconfig (created in a Makefile) so that it can be packed into a -compat package.

Otherwise looks good to me, thank you!